modulus/datapipes/gnn/lagrangian_dataset.py

```diff
@@ -0,0 +1,346 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
Please add unit tests for `lagrangian_dataset.py` and `lagrangian_reading_utils.py`.